src/Components/Modal/index.js

```diff
@@ -7,9 +7,26 @@ import styles from './style.module.scss';
 
 function Modal(props) {
   const {
-    modalOpen, setModalOpen, modalContentRef, children, title,
+    modalOpen,
+    setModalOpen,
+    modalContentRef,
+    children,
+    title,
+    buttonsConfig,
   } = props;
 
+  // Default button configuration if none is provided
```

Comment:
i think your code here speaks for itself ✅, not sure we need this comment (but we can keep it either way, no biggie)

Reply:
Good call...I removed this comment 😀